website/project/views/node.py

```diff
@@ -203,16 +203,25 @@ def project_new_node(auth, node, **kwargs):
                 http.BAD_REQUEST,
                 data=dict(message_long=e.message)
             )
-
+        redirect_url = new_component.url
         message = (
-            'Your component was created successfully. You can keep working on the component page below, '
+            'Your component was created successfully. You can edit contributor permissions below, '
             'or return to the <u><a href="{url}">project page</a></u>.'
         ).format(url=node.url)
+        if form.inheritContributors.data != '':
+            for contributor in node.contributors:
+                new_component.add_contributor(contributor, permissions=node.get_permissions(contributor), auth=auth)
+            new_component.save()
+            redirect_url = redirect_url + 'contributors/'
+            message = (
+                'Your component was created successfully. You can edit the contributor permissions, '
```

Comment:
Let's change "You can edit the contributor permissions," to "You can edit contributor permissions below," to be parallel with the other message.

![screen shot 2016-02-11 at 3 07 42 pm](https://cloud.githubusercontent.com/assets/134455/12989161/0c973a24-d0d2-11e5-883a-4e7453ff453e.png)